packages/backend-common/src/reading/AwsS3UrlReader.ts

```diff
@@ -69,9 +69,9 @@ export function parseUrl(
   const host = parsedUrl.host;
 
   // Treat Amazon hosted separately because it has special region logic
-  if (config.host === 'amazonaws.com') {
+  if (config.host === 'amazonaws.com' || config.host === 'amazonaws.com.cn') {
```

Comment:
Looking at this the domain seems ok: https://aws.amazon.com/blogs/enterprise-strategy/getting-started-with-aws-services-in-aws-china-beijing-region-and-aws-china-ningxia-region/#:~:text=The%20domain%20for%20AWS%20China,%3A%20cn%2Dnorthwest%2D1.